imap/http_caldav_sched.c

```diff
@@ -1390,6 +1393,17 @@ static unsigned propcmp(icalcomponent *oldical, icalcomponent *newical,
     icalproperty *oldprop = icalcomponent_get_first_property(oldical, kind);
     icalproperty *newprop = icalcomponent_get_first_property(newical, kind);
 
+    if ((!oldprop != !newprop) &&
```

Comment:
Switching from no explicit DURATION provided to DURATION: one hour shall not send a notification for DTSTART value:date-time, as in this cases the implicit duration is anyway one hour.

Reply:
I'm not sure that I understand the scenario that you're describing.  Can you be more explicit with before and after examples using DTSTART, DTEND, DURATION

Reply:
My last comment was not very correct.  Changing from
```
DTSTART;VALUE=DATE:20220505
```
to
```
DTSTART;VALUE=DATE:20220505
DTEND;VALUE=DATE:20220506
```
or back, is not a real change for VEVENTS.

Reply:
How can a buggy iCalendar implementation impact a meeting, which content was not changed semantically and therefore has not really changed?

Reply:
I mean that it's safer that all implementations see the exact same time properties, rather than hoping they all agree on what's semantically equivalent. But do not know what motivated these changes and this discussion. Is it that currently, people perceive that too many notifications for semantically equivalent meetings get send out by Cyrus?

Reply:
We (FM) had a user support ticket where the use says they did not change an event, but updates were sent out to attendees.  Apparently their client changed DURATION to DTEND (or vie-versa).  @neilj seems to agree that this should be considered a no-op